packages/webindex/webindex.js

```diff
@@ -0,0 +1,51 @@
+WebIndexData = new Mongo.Collection('webindex_data');
+
+Settings = {
+  downloadInterval: moment.duration({ minutes: 15 }).asMilliseconds(),
+  feedUrl: 'http://thewebindex.org/wp-content/uploads/2012/05/Web_Index_JSON_data_20141.js',
+  limit: 10,
+  map: {
+    width: 450,
+    height: 270,
+    scale: 110,
+    squash: 0.90,
+    bumpDown: 30,
+    bumpLeft: 30
+  }
+};
+
+WebIndexWidget = function(doc) { Widget.call(this, doc); };
+WebIndexWidget.prototype = Object.create(Widget.prototype);
+WebIndexWidget.prototype.constructor = WebIndexWidget;
+
+WebIndex = {
+  widget: {
+    name: 'Web Index',
+    description: 'Shows measures of the Web’s contribution to social, economic and political progress in countries across the world.',
+    url: 'http://thewebindex.org/data/',
+    dimensions: { width: 3, height: 2 },
+    category: 'access',
+    typeIcon: 'globe',
+    constructor: WebIndexWidget
+  },
+  org: {
+    name: 'World Wide Web Foundation',
+    shortName: 'Web Index',
+    url: 'http://thewebindex.org/'
+  },
+  metrics: [
```

Comment:
This is kind of a weird place for this. The WebIndex object is what the dashboard uses to add the widget to the registry, so the `widget` and `org` keys are special. This is also the only object exposed from the package to the outside app. Is there a reason the outside app would need access to this, or can it just go in the widget's own `Settings` object?